examples/iot/policy.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: iot.aws.crossplane.io/v1alpha1
+kind: Policy
+metadata:
+  name: example-policy
+spec:
+  forProvider:
+    region: us-east-1
+    policyName: example-policy
```

Comment:
can we ignore in code-gen policyName and use metadata.name ?
https://github.com/crossplane/provider-aws/blob/master/CODE_GENERATION.md#external-name

Reply:
I ignored the policyName field in code-generator.